shared/src/commonMain/kotlin/co/nimblehq/mark/kmmic/domain/model/AuthToken.kt

```diff
@@ -0,0 +1,22 @@
+package co.nimblehq.mark.kmmic.domain.model
+
+import co.nimblehq.mark.kmmic.data.service.auth.model.AuthTokenResponse
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class AuthToken constructor(
```

Comment:
We can omit this constructor
```suggestion
data class AuthToken(
```

Reply:
@suho updated at [b5c86d3](https://github.com/markgravity/kmm-ic/pull/67/commits/b5c86d34cf43b9e1cac1bdf1dc2ae0f36ddf159a)